packages/drivers/odsp-driver/src/odspDocumentServiceFactoryCore.ts

```diff
@@ -269,7 +269,7 @@ export class OdspDocumentServiceFactoryCore
 
 	protected async createDocumentServiceCore(
 		resolvedUrl: IResolvedUrl,
-		odspLogger: ITelemetryLogger,
+		odspLogger: ITelemetryLoggerExt,
```

Comment:
I think in order to make this not a breaking change, it's preferable to take `ITelemetryBaseLogger` here, and `LocalOdspDocumentServiceFactory` can call `createChildLogger()` inside its override of this function.

Reply:
Seems like this `OdspDocumentServiceFactoryCore` is tagged as @alpha and all objects created by this function are also @alpha. I think it's ok to use the latest logger type `Ext`.

Reply:
Hmm not really, Ext things should ideally be internal, not even "alpha" APIs should use them.

Reply:
I see. Use `Base` type here.